resque-scheduler.gemspec

```diff
@@ -55,6 +55,6 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency 'mono_logger', '~> 1.0'
   spec.add_runtime_dependency 'redis', '>= 3.3'
-  spec.add_runtime_dependency 'resque', '~> 1.26'
+  spec.add_runtime_dependency 'resque', '>= 1.26', '<= 2.0.0'
```

Comment:
Note that this supports 2.0.0 but not 2.0.1 or later

Reply:
Correct, I can only prove that 2.0.0 works at this time.

Reply:
To be fair, by my own logic a major version bump typically contains breaking changes.
Thus since 2.0.0 did not break resque-scheduler, 2.x.x should not either.

In that case, we should change it to `spec.add_runtime_dependency 'resque', '>= 1.26', '< 3'`
`

Reply:
I've changed as per request based on reply by @rafaelfranca